Formula/v2ray.rb

```diff
@@ -20,6 +20,16 @@ class V2ray < Formula
 
   depends_on "go" => :build
 
+  resource "geoip" do
+    url "https://github.com/v2fly/geoip/releases/download/202104080008/geoip.dat"
+    sha256 "1ba724e396f31e89d05f4cc9e912e13c1c5ea4478a2a0901e9cbb9f9a8b9c5db"
+  end
+
+  resource "geosite" do
+    url "https://github.com/v2fly/domain-list-community/releases/download/20210407065022/dlc.dat"
+    sha256 "3fb0dc75a94fafe55445ffd14c2317c8f871d63732170b5c8d5e9d2efcf9c655"
+  end
```

Comment:
How do we know which versions of these to use when updating the formula?

Reply:
Just use the latest version of `geoip.dat` & `geosite.dat`. It doesn't matter.